context/context.js

```diff
@@ -333,6 +355,108 @@ export function FundsProvider({children}) {
   );
 }
 
+export function ArchiveProvider({children}) {
+  const [archive, setArchive] = useState(null);
+  const [shouldReload, setShouldReload] = useState(true);
+  const {activities} = useActivities();
+
+  useEffect(() => {
+    const getArchive = async () => {
+      try {
+        const url = "/api/tasksValues";
+        const response = await fetch(url);
+        if (response.ok) {
+          const data = await response.json();
+          setArchive(data);
+          console.log(archive);
+          setShouldReload(false);
+        } else {
+          throw new Error(
+            `Fetch fehlgeschlagen mit Status: ${response.status}`
+          );
+        }
+      } catch (error) {
+        console.log(error);
+        alert(error.message);
+      }
+    };
+    getArchive();
+  }, [shouldReload]);
+
+  const sendToMongoArchive = async id => {
+    const activityToArchive = activities?.filter(
+      activity => activity.id === id
+    );
+
+    const archiveValue = {
+      title: activityToArchive[0].titel,
+      link: activityToArchive[0].link,
+      blockchain: activityToArchive[0].blockchain,
+      deadline: activityToArchive[0].date,
+      description: activityToArchive[0].description,
+    };
+
+    const options = {
+      method: "POST",
+      headers: {"Content-Type": "application/json"},
+      body: JSON.stringify(archiveValue),
+    };
+    const response = await fetch("api/tasksValues", options);
+    const result = await response.json();
+    setShouldReload(true);
+  };
+
+  const handleDelete = async id => {
+    try {
+      const response = await fetch(`/api/tasks/${id}`, {
+        method: "DELETE",
+      });
+      console.log(id);
```

Comment:
Please delete this `console.log`. In your final PR there should be no console logs you are just using for development.

Reply:
done!